Source/WebCore/Modules/modern-media-controls/controls/button.js

```diff
@@ -150,6 +150,7 @@ class Button extends LayoutItem
 
     handleEvent(event)
     {
+        event.stopPropagation();
```

Comment:
should we limit this to just the pause event ?

Reply:
This would only be events that directly targeted this button, like "click", "mouseover", etc.